modules/nf-core/sylph/profile/tests/main.nf.test

```diff
@@ -0,0 +1,79 @@
+nextflow_process {
+
+    script "../main.nf"
+    process "SYLPH_PROFILE"
+    tag "modules"
+    tag "modules_nfcore"
+    tag "sylph"
+    tag "sylph/profile"
+
+    test("sarscov2 illumina single-end [fastq_gz]") {
+        when {
+            process {
+                """
+                input[0] = [ [ id:'test', single_end:true ], // meta map
+                            [ file(params.test_data['sarscov2']['illumina']['test_1_fastq_gz'], checkIfExists: true) ]
```

Comment:
Can you swap these to the more recent file paths please

Reply:
I updated the file paths.